class.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'my_enumerable_module'
+
+class MyList
+  include MyEnumerable
+  def initialize(list)
+    @list = list
+  end
```

Comment:
- [ ] You should not limit here to only one argument, because when I try to add this
` list = MyList.new(1, 2, 3, 4)` I get this error 
![image](https://user-images.githubusercontent.com/62705947/222460763-723e123e-4766-46ad-874d-9bfd106ea5f0.png)
Instead make this as `def initialize(*list)`

Reply:
Sure, thanks!